pkg/operator/operator.go

```diff
@@ -160,6 +160,14 @@ func Start(ctx context.Context, buildInfo starboard.BuildInfo, operatorConfig et
 		}).SetupWithManager(mgr); err != nil {
 			return fmt.Errorf("unable to setup vulnerabilityreport reconciler: %w", err)
 		}
+
+		if err = (&controller.TTLReportReconciler{
```

Comment:
It's currently inside the if `operatorConfig.VulnerabilityScannerEnabled` so that is already checked. But we could ask our self if that is a good thing or not. For now it works but if we want the controller to manage multiple reports in the future we will need to move it out.
But it's a bit of pre-optimization to do that and for now I think we can skip it.